Gemfile

```diff
@@ -27,7 +27,7 @@ end
 
 group :test do
   gem 'activerecord'
-  gem 'sqlite3', platform: :ruby
+  gem 'sqlite3', platform: [:ruby, :mingw, :x64_mingw, :mswin]
```

Comment:
They are using different compiler, (mswin using Visual C++), (mingw using GCC) but should be included in all mswin, so writing mswin should enough here, although I don't known why Rails [still using all of them](https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/Gemfile#L56), guess the history reason of rubygems, but not check yet.